Base/QTGUI/qSlicerWebWidget.cxx

```diff
@@ -101,16 +117,30 @@ void qSlicerWebWidgetPrivate::init()
 #else
   this->WebView = new qSlicerWebEngineView();
 
+  QSettings settings;
+  bool developerModeEnabled = settings.value("Developer/DeveloperMode", false).toBool();
+  if (developerModeEnabled)
+    {
+    // Enable dev tools by default for the test browser
+    if (qgetenv("QTWEBENGINE_REMOTE_DEBUGGING").isNull())
+      {
+      qputenv("QTWEBENGINE_REMOTE_DEBUGGING", "1337");
```

Comment:
I think this is arbitrary. See https://doc.qt.io/qt-5/qtwebengine-debugging.html

Reply:
Yes, I just followed the example.  Anyone can set a different port via the environment.

Reply:
For reference the Qt test is here: https://code.woboq.org/qt5/qtwebengine/tests/quicktestbrowser/main.cpp.html#66

(Small aside, that woboq site is pretty handy for browsing Qt code, in case you haven't already seen it).

Reply:
Interesting to know. I used [OpenGrok](https://github.com/oracle/opengrok) in the past for such purposes, but [woboq claims to be different](https://woboq.com/codebrowser.html#how): 

> In contrast to other solutions (LXR, OpenGrok) it semantically analyzes the code as a compile step.

(whatever "semantically analyzes" means!)

Reply:
The selftest module includes a button for bringing up the web inspector and inspection is enabled (only) when you are in Developer mode or when you set the environment variable yourself, so I think it's good enough for now.

I am willing to write more documentation for this.  I guess the rst-user-and-dev-guide branch is the right place, but that can't be part of this PR.

Reply:
> The selftest module includes a button for bringing up the web inspector

OK, that's even better than documentation.